src/scene/layer/editablePolygon.tsx

```diff
@@ -50,6 +50,7 @@ export interface IPolygon {
   type: PolygonType,
   verticies: Array<Vector2d>
   visibleOnTable: boolean;
+  opacity: number;
```

Comment:
opacity may be undefined since it wasn't there at the beginning...

Reply:
It's always set to one by default when you create a new poly

Reply:
Ah, on pre-existing polys that have already been created! Yep, I'll handle that!